daemon/hostdaemon.go

```diff
@@ -82,18 +112,96 @@ func (d *HostDaemon) connectWithRetry() {
 	conn, err := grpc.Dial(fmt.Sprintf("%s:%d", d.addr, d.port), grpc.WithTransportCredentials(insecure.NewCredentials()), grpc.WithDefaultServiceConfig(retryPolicy))
 	if err != nil {
 		d.log.Error(err, "did not connect")
+		return err
 	}
 	d.conn = conn
 	d.client = pb.NewBridgePortServiceClient(conn)
+	return nil
+}
+
+func (d *HostDaemon) cniCmdAddHandler(req *cnitypes.PodRequest) (*cni100.Result, error) {
+	pf := 0
+	vf := req.CNIConf.VFID
+	mac := req.CNIConf.RuntimeConfig.Mac
+	d.log.Info("addHandler", "CNIConf", req.CNIConf)
```

Comment:
I think we can remove these prints.

Reply:
Are you sure? last time I removed logs, the first thing we ended up doing during debug sessions is put the logs back.

Reply:
I think the VLAN one is redundant. But very true for CNIConf